libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -227,16 +227,31 @@ void AP_Baro::calibrate(bool save)
    update the barometer calibration
    this updates the baro ground calibration to the current values. It
    can be used before arming to keep the baro well calibrated
+
+   The altitude argument is an optional argument holding the current absolute
+   altitude of the vehicle. If it is set to 0, then we assume there is no
```

Comment:
Yeah, I like the libraries to has independent from each other as possible, but I can see how the code duplication was a pain.